samples/lj-gc.sxx

```diff
@@ -14,11 +18,16 @@ probe @pfunc(ngx_process_events_and_timers)?, timer.profile
         }
 
         if (L == 0) {
-            printf("Failed to get the main Lua VM\n")
-            exit()
-        }
+            if(retries > @RETRIES_THRESHOLD) {
```

Comment:
```suggestion
            if (retries > @RETRIES_THRESHOLD) {
```

Reply:
Also, need to double check and make sure OpenResty was compiled with debug on, otherwise optimization may cause strange runtime behavior.